docs/software/communication/openmpi.md

```diff
@@ -6,5 +6,39 @@ However, [OpenMPI](https://www.open-mpi.org/) can be used as an alternative in s
 
 To use OpenMPI on Alps, it must be built against [libfabric][ref-communication-libfabric] with support for the [Slingshot 11 network][ref-alps-hsn].
 
+## Using OpenMPI
+
+!!! warning
+    Building and using OpenMPI on Alps is still [work in progress](https://eth-cscs.github.io/cray-network-stack/).
+    The instructions found on this page may be inaccurate, but are a good starting point to using OpenMPI on Alps.
+
+!!! todo
+    Deploy experimental uenv.
+
 !!! todo
-    Building OpenMPI for Alps is still work in progress: https://eth-cscs.github.io/cray-network-stack/.
+    Document OpenMPI uenv next to prgenv-gnu, prgenv-nvfortran, and linalg?
+
+OpenMPI is provided through a [uenv][ref-uenv] similar to [`prgenv-gnu`][ref-uenv-prgenv-gnu].
+Once the uenv is loaded, compiling and linking with OpenMPI and libfabric is transparent.
+At runtime, some additional options must be set to correctly use the Slingshot network.
+
+First, when launching applications through slurm, [PMIx](https://pmix.github.com) must be used for application launching.
+This is done with the `--mpi` flag of `srun`:
+```bash
+srun --mpi=pmix ...
+```
+
+Additionally, the following environment variables should be set:
+```bash
+export PMIX_MCA_psec="native" # (1)
+export FI_PROVIDER="lnx" # (2)
+export FI_LNX_PROV_LINKS="shm+cxi" # (3)
+export OMPI_MCA_pml="^ucx" # (4)
+export OMPI_MCA_mtl="ofi" # (5)
```

Comment:
These are optional in my experience when using just OFI CXI. I mostly used them when experimenting with LINKx and I tried to replicate as close as possible the cmd line from the LINKx paper (i.e. the one from LANL/ORNL about OpenMPI on Cray EX).

Reply:
So what you're saying is: if OpenMPI is "correctly" built for Alps (only the necessary providers/backends/etc.) then these two should not be needed? If yes, I agree, we can leave this out.